Documentation/Helm-Charts/operator-chart.md

```diff
@@ -41,96 +41,96 @@ For example settings, see the next section or [values.yaml](https://github.com/r
 
 The following tables lists the configurable parameters of the rook-operator chart and their default values.
 
-| Parameter                           | Description                                                                                                                                      | Default                                                   |
```

Comment:
Too many lines have changed, 

try editing with normal text editor, to preserve other lines.

Reply:
please fix this one. @yati1998